docs/future-work/environment-improvements/object-behavior-test-bed.md

```diff
@@ -37,3 +37,7 @@ We can test Monty's ability to model object behaviors with successively more dif
 - Supervision during learning -> provide object ID, pose & state in sequence at each step.
 - Evaluate accuracy & speed of recognizing behavior & morphology ID, pose & state (+visualize).
 - Could start with 2D environment to make the state dimension easier to visualize (keep in mind potential difficulty in extracting movement from a 3D depth image).
+
+## Existing Test Environment
+An early version of a simple object behavior environment can be found [here](https://github.com/thousandbrainsproject/monty_lab/tree/main/object_behaviors). It can be used to simulate physical joints and springs and retrieve locations and orientations in CMP format. See a simple example of a stapler below.
```

Comment:
```suggestion
An early version of a simple object behavior environment can be found [here](https://github.com/thousandbrainsproject/monty_lab/tree/main/object_behaviors). It can be used to simulate physical joints and springs and retrieve locations and orientations in CMP format. See a simple example of a "stapler"/hinge-object below.
```